src/faldbt/project.py

```diff
@@ -42,12 +42,19 @@ class FalGeneralException(Exception):
 
 
 @dataclass
-class DbtTest:
-    node: Any
+class _DbtNode:
```

Comment:
Is class names with _ common practice?

Reply:
i've seen a couple times. It's a way to communicate that a class is not to be imported anywhere else.